docs/components/modeler/bpmn/bpmn-primer.md

```diff
@@ -114,12 +114,12 @@ The following is an introduction to BPMN 2.0, its elements, and their execution
 
 ## Modeling BPMN diagrams
 
-The best tool for modeling BPMN diagrams for Zeebe is **Camunda Modeler**.
+The best tool for modeling BPMN diagrams for Zeebe is **Desktop Modeler**.
```

Comment:
Why not just link here to our "Modeler" page. Then the user can decide to either use the Desktop or the Web version.

Reply:
+1 good catch

Reply:
https://github.com/camunda-cloud/camunda-cloud-documentation/pull/620/commits/63bbeccadb67aa89cc4ee4b22e678e67a21a2d55